uint32 seq		# Image sequence number

uint32 ORB_QUEUE_LENGTH = 3

